README.md

```diff
@@ -3,41 +3,34 @@
 # Particle Accelerator Lattice Standard (PALS)
 
 An effort to create a standard to promote lattice information exchange for particle accelerators.
-Associated PALS code will be usable in a program to read/write lattice files.
+PALS can also serve as the language for new simulation programs and 
```

Comment:
Language seems a bit vague here, how about:
```suggestion
PALS can also serve as the beamline scheme for new simulation programs and 
```

Reply:
I think language format is not a commonly used name, the computer science term of what we do is schema or markup.

Reply:
Common or not, I find `language format` to be clearer than `schema` or `markup`

Reply:
Thanks, agreed.

Following these definitions, I think **schema** or **data model** is what we define here.